packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -156,27 +155,15 @@ export class MediaPlaceholder extends Component {
 			const isImage = isOneType && 'image' === allowedTypes[ 0 ];
 			const isVideo = isOneType && 'video' === allowedTypes[ 0 ];
 
-			if ( ! instructions ) {
-				if ( hasUploadPermissions ) {
-					instructions = __( 'Drag a media file, upload a new one or select a file from your library.' );
+			if ( ! instructions && mediaUpload ) {
+				instructions = __( 'Drag a media file, upload a new one or select a file from your library.' );
 
-					if ( isAudio ) {
-						instructions = __( 'Drag an audio, upload a new one or select a file from your library.' );
-					} else if ( isImage ) {
-						instructions = __( 'Drag an image, upload a new one or select a file from your library.' );
-					} else if ( isVideo ) {
-						instructions = __( 'Drag a video, upload a new one or select a file from your library.' );
-					}
-				} else if ( ! hasUploadPermissions && onSelectURL ) {
```

Comment:
The fallback instructions here are not that useful IMO and the "role" is not the only reason the user can't upload. In a serverless gutenberg instance, upload is never allowed for instance.

Reply:
Hi @youknowriad, I am not sure if the design will look good if the placeholder does not contain a message.
Maybe we can use a simple message like:
"Link to an audio file, using the options bellow".

Reply:
Actually, the design felt better to me without any instructions as there was only a unique action.
<img width="744" alt="Capture d’écran 2019-03-21 à 6 22 27 PM" src="https://user-images.githubusercontent.com/272444/54771897-89fbb500-4c06-11e9-99cf-98107ad0468d.png">

Reply:
I do like the simplicity of the single button action. 

One concern is that a user might be versed in WordPress and be an admin on one site, and a contributor on another site. This user might be confused as to why the Upload button is gone here, and a description would mitigate that. It also somewhat helps explain this image placeholder, which is a somewhat new flow compared to traditional editors — here you insert the block first, then pick the image. 

At the same time, it is extremely likely that this image placeholder will go through a number of additional iterations, such as making it resizable like an image, or allow you to show the media library inline for one-click image picking. In that vein it doesn't seem too fruitful to hold up solid improvements to hyper-optimize and edgecase for an interface that is going to change anyway. 

In other words: descriptions are helpful, but okay to go with this for now given it will be revisited.